### ISSUE TITLE
39% brightness H6159

### ISSUE THREAD

**Issue by dallombra** [bwp91/homebridge-govee#37]
Hi, I have two H6159 led strips and I can’t set them to full brightness.

i’m running the latest version on the plug-in on docker on synology hardware.

Thanks!

**Comment by bwp91** [comment]
Hi @dallombra 

Please turn on plugin `debug` mode and use the Govee app to change the brightness of your device. Does the value in the logs for the brightness change? correctly? You'll see entries like this where you can check:

```
received update [{"online":true,"powerState":"off","brightness":44,"color":{"r":195,"b":60,"g":123}}].
```

Also, if you use the Home app to change brightness, is the value updated correctly?

Thanks

**Comment by CyberRad** [comment]
I have the H6159 and see this as well. I did as you suggested and I am seeing that 100% in the Govee app reports as 39% from the plugin output.
```
[1/23/2021, 20:45:38] [Govee] [Val] received update [{"online":true,"powerState":"on","brightness":100,"color":{"r":181,"b":74,"g":122}}].
[1/23/2021, 20:45:38] [Govee] [Val] current state [on].
[1/23/2021, 20:45:38] [Govee] [Val] current brightness [39%].
```
When I try to bump it up in homekit I see the following output:
```
[1/23/2021, 20:47:14] [Govee] [Val] sending update {"name":"brightness","value":188}.
[1/23/2021, 20:47:14] [Govee] [Val] could not be updated and its status will revert soon - Request failed with status code 400.
```

**Comment by bwp91** [comment]
Thanks @CyberRad 

The reason why I asked for this extra info is because the H6159 is the model that I own and so find it peculiar that you experience this behaviour when I don't.

From the Home app if you long-click on the accessory and scroll down to view the information (Manufacturer, Serial Number, ...) what _exactly_ does it say for `Model`?

**Comment by CyberRad** [comment]
H6195
![IMG_1017](https://user-images.githubusercontent.com/3980254/105628727-a40f0b80-5e0c-11eb-870d-de54ae2eaafd.PNG)

**Comment by bwp91** [comment]
Ah! You have the H6195 not the H6159

**Comment by bwp91** [comment]
I'll get a fix out shortly

**Comment by CyberRad** [comment]
You are right. I wonder if that is the original posters issue as well. Thanks!

**Comment by bwp91** [comment]
Brightness fix for the **H6195** is in v2.8.3 :)

Don't know whether to close this issue as the original post was about the H6159

**Comment by CyberRad** [comment]
I can confirm that this has fixed the issue with the H6195. Thanks
